changelogs/unreleased/8052-agent-cache-improve-naming.yml

```diff
@@ -0,0 +1,3 @@
+description: "Clarify user facing interface."
+change-type: patch
+destination-branches: [master]
```

Comment:
As far as I can see we don't have a changelog entry yet for the changes to the cache. So we need to add one.